internal/security/fileprovider/provider_test.go

```diff
@@ -28,9 +28,11 @@ import (
 	"github.com/edgexfoundry/edgex-go/internal/security/fileprovider/config"
 	"github.com/edgexfoundry/edgex-go/internal/security/secretstoreclient"
 	. "github.com/edgexfoundry/edgex-go/internal/security/secretstoreclient/mocks"
-	. "github.com/edgexfoundry/go-mod-bootstrap/security/authtokenloader/mocks"
-	. "github.com/edgexfoundry/go-mod-bootstrap/security/fileioperformer/mocks"
+
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	. "github.com/edgexfoundry/go-mod-secrets/pkg/token/authtokenloader/mocks"
```

Comment:
Blank line between go-mod-core-contracts and go-mod-secrets.

Reply:
I thought we were grouping imports by the following:

1. Standard Go packages
2. Packages within the same repo
3. EdgeX packages in different repos 
4. All other packages from external modules

Is this not the case?

Reply:
I've tried to consistently enforce:

```
standard go packages
<empty line>
edgex-go-based packages
<empty line>
other edgex repository packages -- each repository's packages separated from other repository's packages by an <empty line>
<empty line>
other github.com-based non-edgex repository packages (all grouped together).
```